packages/docs/components/rating-docs-demo/RatingColorDocsDemo.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <div class="rating-container">
+      <va-rating v-model="value" color="success" />
+      <va-rating v-model="value" color="info" />
+      <va-rating v-model="value" color="danger" />
+      <va-rating v-model="value" color="warning" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import VaContent from '../../../ui/src/components/vuestic-components/va-content/VaContent.vue'
+import VaRating from '../../../ui/src/components/vuestic-components/va-rating/VaRating.vue'
+
+@Component({
+  components: { VaContent, VaRating },
+})
+export default class RatingColorDocsDemo extends Vue {
+  data () {
+    return {
+      value: 3,
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Assume user will see all these styles and consider them example usage.
Do you think that works for us?

Reply:
I think the user will figure it out, he is a frontend developer. But we can remove styles if you think this is more correct.

Reply:
@SergeiPleshkov the proper approach for user, except for some exotic cases, would be to use global styles we provide.
You can check refs and see how they handle common cases. Normally you won't see any css at all.